src/ruby/ext/grpc/extconf.rb

```diff
@@ -63,19 +63,10 @@
 ENV['CFLAGS'] = '-DGPR_BACKWARDS_COMPATIBILITY_MODE'
 
 output_dir = File.expand_path(RbConfig::CONFIG['topdir'])
-grpc_lib_dir = File.join(output_dir, 'libs', grpc_config)
 ENV['BUILDDIR'] = output_dir
 
-unless windows
```

Comment:
This won't work because it breaks the windows build (which builds the core library in a  separate, earlier step